.eslintrc.js

```diff
@@ -1,10 +1,11 @@
 module.exports = {
-  extends: '@mate-academy/eslint-config',
+  extends: "@mate-academy/eslint-config",
   env: {
-    jest: true
+    jest: true,
   },
   rules: {
-    'no-proto': 0
+    "no-proto": 0,
+    quotes: "off",
```

Comment:
The `quotes` rule is set to "off" as a string, but it should be set as a number (0, 1, or 2) or an object. Consider changing it to `0` to turn it off, or configure it properly if you intend to enforce a specific style.